code-bar-job-board-app/src/components/Footer/index.js

```diff
@@ -0,0 +1,62 @@
+
+
+const Footer = () => {
+
+    
+
+    return (
+        <footer style={{backgroundColor: '#206CC2'}}>
+            <div>
+                <div className='footer-info'>
+                    <p>&#169; 2021 codebar</p>
```

Comment:
At some point could we make the year dynamic. So create a date variable and use the javascript Date function. Saves us having to update it every year ha.